contracts/contracts/ocr2/aggregator.cairo

```diff
@@ -1020,10 +1023,14 @@ func has_billing_access{
     end
 
     let (access_controller) = billing_access_controller_.read()
+    let empty_data_len = 0
+    let (empty_data) = alloc()
 
     let (has_access: felt) = IAccessController.has_access(
         contract_address=access_controller,
-        address=caller
+        user=caller,
+        data_len=empty_data_len,
+        data=empty_data
     )
     return (has_access)
```

Comment:
1. `IAccessController.check_access` is available and does exactly this, checks against an empty calldata.
2. `has_billing_access` returns a boolean, but is used across the `aggregator` contract as an access modifier. As the fn is `require_billing_access`, but it is not. It will not error, but just return true/false which is currently not handled.

Reply:
1. `check_access` in current form panics if one doesn't have an access.
2. I think regarding implementation details of `aggregator.cairo` @archseer can answer better.

Reply:
@archseer Shall we address this in this PR or create an issue for future fix?  (Discussed in dm on slack)

Reply:
Fixed in the e2e PR